sources/packages/api/src/route-controllers/application/models/application.model.ts

```diff
@@ -103,3 +106,46 @@ export interface ApplicationWithProgramYearDto {
   programYearId: number;
   active: boolean;
 }
+
+export interface SuccessWaitingStatus {
+  success: boolean;
+  waiting: boolean;
+}
+
+export interface ApplicationIncomeVerification {
```

Comment:
Besides these ones must also be class to be visible in swagger and also all the classes must be present in a file suffixed with .dto.ts for swagger to recognize them as dto.